helpers/helper-translation-discussion.php

```diff
@@ -524,7 +529,15 @@ function gth_discussion_callback( WP_Comment $comment, array $args, int $depth )
 	<?php endif; ?>
 	</div>
 	<?php endif; ?>
-	<div class="comment-content" dir="auto"><?php comment_text(); ?></div>
+	<div class="comment-content" dir="auto">
+		<?php comment_text(); ?>
+		<?php if ( $reject_reason ) : ?>
+		<p>
+			<?php echo esc_html( _n( 'Rejection Reason: ', 'Rejection Reasons: ', count( $reject_reason ) ) ); ?>
```

Comment:
I've used `_n` here for showing plural instead of having a `(s)` in the text.

Reply:
> I've used `_n` here for showing plural instead of having a `(s)` in the text.

Oh nice, thanks!